lib/clog.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "json"
+
+class Clog
+  def self.emit(msg)
+    out = if block_given?
+      case v = yield
+      when Hash
+        v
+      else
+        {invalid_type: v.class.to_s}
+      end
+    else
```

Comment:
I think in a future patch this can be even smarter to accept an Array that can contain both Sequel model instances and Hashes and have a reasonable automatic/idiomatic behavior. But maybe for next time. For now I unpack sequel models like: `{strand: strand.values}` or `{github_runner: github_runner.values}`